docs/_docs/themes.md

```diff
@@ -71,13 +149,29 @@ For example, if your theme has a `/_layouts/page.html` file, and a page has `lay
 
 ### Assets
 
-Any file in `/assets` will be copied over to the user's site upon build unless they have a file with the same relative path. You may ship any kind of asset here: SCSS, an image, a webfont, etc. These files behave just like pages and static files in Jekyll: if the file has [YAML front matter]({{ site.baseurl }}/docs/frontmatter/) at the top, then it will be rendered. If it does not have YAML front matter, it will simply be copied over into the resulting site. This allows theme creators to ship a default `/assets/styles.scss` file which their layouts can depend on as `/assets/styles.css`.
+Any file in `/assets` will be copied over to the user's site upon build unless they have a file with the same relative path. You can ship any kind of asset here: SCSS, an image, a webfont, etc. These files behave just like pages and static files in Jekyll:
+
+* If the file has [YAML front matter](../docs/frontmatter/) at the top, it will be rendered.
+* If the file does not have YAML front matter, it will simply be copied over into the resulting site.
+
+This allows theme creators to ship a default `/assets/styles.scss` file which their layouts can depend on as `/assets/styles.css`.
 
 All files in `/assets` will be output into the compiled site in the `/assets` folder just as you'd expect from using Jekyll on your sites.
 
 ### Stylesheets
 
-Your theme's stylesheets should be placed in your theme's `/_sass` folder, again, just as you would when authoring a Jekyll site. Your theme's styles can be included in the user's stylesheet using the `@import` directive.
+Your theme's stylesheets should be placed in your theme's `_sass` folder, again, just as you would when authoring a Jekyll site.
+
+```
+  _sass
+├── jekyll-theme-my-awesome-theme.scss
+```
+
+Your theme's styles can be included in the user's stylesheet using the `@import` directive.
+
+```css
+{% raw %}@import "{{ site.theme }}";{% endraw %}
```

Comment:
while I'm :+1:  on the use of `@import "{{ site.theme }}";`, lets hope it doesn't create confusion or an extra step of customization for people opting to not using gem-based themes or in the rare situation that the theme-gem doesn't name their main sass file the same as their theme-name..